notes/v1.0.0-beta1.md

```diff
@@ -0,0 +1,211 @@
+# Changelog
+
+Release binaries can be downloaded with the following links:
+
+<https://download.elastic.co/downloads/ecctl/1.0.0-beta1/ecctl_1.0.0-beta1_darwin_amd64.tar.gz>
```

Comment:
The links need an extra newline between them so they show up in different lines (markdown is annoying 😄) 

![Screen Shot 2019-12-23 at 10 45 32 AM](https://user-images.githubusercontent.com/7670244/71327774-83251880-2571-11ea-88c5-611aa66d77e8.png)